pkg/antctl/antctl.go

```diff
@@ -198,6 +199,34 @@ var CommandList = &commandList{
 			commandGroup:        get,
 			transformedResponse: reflect.TypeOf(podinterface.Response{}),
 		},
+		{
+			use:     "ovsflows",
+			aliases: []string{"of"},
+			short:   "Dump OVS flows",
+			long:    "Dump the OVS flows installed for the specified entity. Right now only flows for a Pod is supported.",
```

Comment:
s/Right now only flows for a Pod is supported./As of now, only Pod flows are supported.

Reply:
My last update also supports dumping all flows on the Node if no extra parameter is provided. So, revised the description to reflect the change. Please check.

Reply:
Looks good